### PR TITLE
Import repo skeleton from bundle template

### PR DESCRIPTION
**Pull request by shayancanonical**
## Issue
We created an empty repository and have to import a lot of the bundle skeleton from Jarred's template repo and also from the postgresql-bundle repos.

## Solution
Import skeleton files

## Release Notes
Import repo skeleton from bundle template

### CODE REVIEW

File: .github/workflows/on_bundle_update_available.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          signoff: false
+          branch: update-bundles
+          delete-branch: true
+          title: 'Update Bundles'
+          body: |
+            Update bundles with new revisions from Charmhub.
+          labels: |
+            automated pr
+          assignees: team-reviewers, shayancanonical, paulomach
+          reviewers: team-reviewers, shayancanonical, paulomach
```
Comment (WRFitch) [inline .github/workflows/on_bundle_update_available.yaml:32]: Remember you're going to need to add a charmhub token and a github PAT to this repo. I don't think you can self-review PRs, so if you're using a PAT from your own account, you need to remove yourself as a reviewer. Don't worry, you'll still get daily notifications ;)

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+deploy: build
+	juju deploy $(BUILD_DIRECTORY)/mysql-k8s-bundle.zip --trust
+
+destroy-model:
+	juju destroy-model --force --destroy-storage $(MODEL_NAME)
+
+# TODO: Convert the echo statements into real statements when ready to release
+release:
+	echo "charmcraft upload $(BUILD_DIRECTORY)/*.zip"
+	echo "charmcraft release mysql-k8s-bundle --revision $(shell charmcraft revisions mysql-k8s-bundle | awk 'NR==2 {print $$1}') --channel=latest/edge"
```
Comment (WRFitch) [inline Makefile:24]: Nice

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+ops >= 1.5.2
```
Comment (WRFitch) [inline requirements.txt:1]: Consider updating to v1.5.3 explicitly

**Review by WRFitch — APPROVED** [review]

**Review by MiaAltieri — APPROVED** [review]

File: releases/latest/mysql-k8s-bundle.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+bundle: kubernetes
+name: mysql-k8s-bundle
+type: bundle
+applications:
```
Comment (marceloneppel) [inline releases/latest/mysql-k8s-bundle.yaml:4]: Just a note: a PR will be automatically created to order those fields alphabetically as described on https://github.com/canonical/postgresql-bundle/pull/8#discussion_r984853762.

**Review by marceloneppel — APPROVED** [review]
Looks great!

File: .github/workflows/on_bundle_update_available.yaml (hunk 1 above)
Comment (shayancanonical) [reply .github/workflows/on_bundle_update_available.yaml]: Great call! Created and added the tokens to the repository secrets

File: requirements.txt (hunk 3 above)
Comment (shayancanonical) [reply requirements.txt]: updated in c7c870362d33e40e70c1d2ff64400dc4c6e33b1b

File: releases/latest/mysql-k8s-bundle.yaml (hunk 4 above)
Comment (shayancanonical) [reply releases/latest/mysql-k8s-bundle.yaml]: good catch! sorted alphabetically in c7c870362d33e40e70c1d2ff64400dc4c6e33b1b

**Review by WRFitch — APPROVED** [review]

**Pull request merged**